client/src/plus/offer-overview/offer-hero/index.scss

```
@@ -67,11 +67,13 @@ $text-stroke-width: 2px;
         }
 
         .button-primary {
+          border: 2px solid var(--button-primary-border-white);
           background-color: var(--text-primary);
```

Probably it makes sense to order these alphabetically:

```suggestion
          background-color: var(--text-primary);
          border: 2px solid var(--button-primary-border-white);
```